src/compositor/lipstickcompositorwindow.cpp

```diff
@@ -292,6 +292,8 @@ void LipstickCompositorWindow::wheelEvent(QWheelEvent *event)
     QWaylandSurface *m_surface = surface();
     if (m_surface) {
         QWaylandSeat *inputDevice = m_surface->compositor()->seatFor(event);
+        QPointF pos(0, 0);
```

Comment:
Could you guard this event by a check for `view != inputDevice->mouseFocus` ?https://github.com/qt/qtwayland/blob/135a3e39c349658e10329ff3988434d0d9b01ce8/src/compositor/compositor_api/qwaylandpointer.cpp#L253

My concern is that if the cursor is currently at a certain position in the window (hovering over some element for example) and the user turns the rotary input, it'd lead to the mouse moving somewhere else for no reason.

Practically speaking, this shouldn't be such a big issue on our smartwatches but I think I would sleep better at night knowing that this situation can't happen.

Also, since we are at it, it's probably worth having a short comment explaining why this is here. Even though we plan to ditch lipstick someday, I suppose it can't hurt.